bin/index.js

```diff
@@ -41,14 +47,18 @@ const writeModifiedRules = ruleIdsArr => {
  * @param {array} results - Array of files with ESLint report for each file
  */
 const disableESLintIssues = results => {
+  const includeFixable = argv.fixable;
```

Comment:
Can we call this option `disable-fixable` instead (for both argument and the constant)? 

```suggestion
  const disableFisable = argv['disable-fixable'];
```

Positive check here 
https://github.com/akonchady/disable-eslint-issues-cli/pull/4/files#diff-9ea21889e9fbc7d82e7dabb0fb280808R61

would be easier to grok as well.

Reply:
@akonchady We could. Which do you think makes more sense as a default? I felt like most users would not want to dirty up their code with disable comments for fixable rules, so I decided we should not disable fixable rules by default, requiring positive opt-in with the `--fixable` flag (or could have named it `--include-fixable`). What do you think?

Reply:
Fair point. `--include-fixable` sounds good to me 👍

Reply:
@akonchady I have updated the option to `--include-fixable` (aliased as `-f`). I have added description text for the new `--help` flag that we support now that this moves us to use `yargs` for command-line parsing. I've updated the README with these new features and options.